### PR TITLE
Add a new command 0x08

### PR DESCRIPTION
**Pull request by bubbledevteam**
Add a new command 0x08(implemented in Bubble FW v2.6 and upper) which request decrypted Libre2/US 14days raw data from Bubble, the data Structure is the same as Libre1,make xdrip+ algorithm work with all Libre type sensors(Libre Pro is not included yet).

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java [hunk 1]
```diff
@@ -31,14 +31,25 @@ public static boolean isBubble() {
         if (activeBluetoothDevice == null || activeBluetoothDevice.name == null) {
             return false;
         }
-        return activeBluetoothDevice.name.contentEquals("Bubble");
+        return activeBluetoothDevice.name.contains("Bubble");
```
Comment (tolot27) [inline app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java:34]: Maybe, `startsWith` is more appropriate here.

**Review by tolot27 — APPROVED** [review]
Looks good to me.

File: app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java (hunk 1 above)
Comment (parapenT1sta) [reply app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java:34]: Thank you!

**Review by jamorham — APPROVED** [review]
This looks okay to me, but I have no knowledge of the hardware changes it addresses. I can only assume that it has been properly tested.

**Pull request merged**